### PR TITLE
Feat create curriculum page 142915151

### PR DESCRIPTION
**Pull request by andela-cnwosu**
Add dummy content and styling for curriculum page.

curriculum page

### CODE REVIEW

File: src/js/components/curriculum/CurriculumPage.js [hunk 1]
```diff
@@ -5,10 +5,66 @@ class CurriculumPage extends React.Component {
...
+                            <div className="cur-links">
+                              <a target="_blank" href="http://coderdojowestcork.files.wordpress.com/2012/05/ping-pong-game_tutorial.pdf">Ping Pong Game: Step-by-Step Tutorial</a>
+                            </div>
+                          </td>
+                      </tr>
+                  </tbody>
+                </table>
+              </div>
+            </div>
+          </div>
```
Comment (andela-iamadi) [inline src/js/components/curriculum/CurriculumPage.js]: In addition, within the row, the part to hold objectives be a separate component. Same could apply markup for resources and Projects so we have something like:

```
<table>
   <tableRow>
      <Objectives />
      <Resources />
      <Projects />
   </TableRow>
<Table />
```

Comment (andela-cnwosu) [inline src/js/components/curriculum/CurriculumPage.js]: Okay. I will break them down. Thanks

File: src/js/components/curriculum/CurriculumBody.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <td>
+              <Objectives
+                title={curriculumList[key].objectives.title}
+                values={curriculumList[key].objectives.values} />
+              <Resources
+                title={curriculumList[key].resources.title}
+                values={curriculumList[key].resources.values} />
+              <Projects
+                title={curriculumList[key].projects.title}
+                values={curriculumList[key].projects.values} />
```
Comment (andela-iamadi) [inline src/js/components/curriculum/CurriculumBody.js]: ❤️

File: src/js/components/curriculum/curriculumList.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    },
+    projects: {
+      title: "Introduction to Scratch II",
+      values: [
+        "Create a Pong Game"
+      ]
+    }
+  }
+}
+
```
Comment (andela-iamadi) [inline src/js/components/curriculum/curriculumList.js]: This is amazing @andela-cnwosu Nice work!

**Pull request merged**